build/VSL.Imports.Closed.targets

```diff
@@ -70,8 +70,8 @@
           Inputs="@(IntermediateAssembly)"
           Outputs="@(IntermediateAssembly);$(PostCompileBinaryModificationSentinelFile)">
     
-    <Exec Command="$(VSLOutDir)\FakeSign.exe &quot;@(IntermediateAssembly)&quot;" />
-    
+    <Exec Condition=" '$(OS)' == 'Windows_NT'" Command="$(VSLOutDir)\FakeSign.exe &quot;@(IntermediateAssembly)&quot;" />
+    <Exec Condition=" '$(OS)' != 'Windows_NT'" Command="mono $(VSLOutDir)\FakeSign.exe &quot;@(IntermediateAssembly)&quot;" />
```

Comment:
There is no straightforward way that I'm aware of. Long term I prefer @jaredpar's suggestion of a `NetExec` task.